README.md

```diff
@@ -37,7 +37,7 @@ If using the console version, run ./tgs.bat in the root of the installation dire
 
 #### Linux (Native)
 
-We recommend using Docker for Linux installations, see below. The content of this parent section may be skipped if you choose to do so.
+It is recommended to run TGS using a systemd script. An example script is provided in the base directory, and assumes that TGS is installed under /opt/tgs
```

Comment:
I still can't recommend this because it can cause unforseen issues that are already accounted for in the Docker version

Reply:
```suggestion
An example systemd script is provided in the base directory, which assumes that TGS is installed under `/opt/tgs`
If you are experiencing issues using systemd to host TGS and have confirmed all requisites are installed, consider using the docker setup seen below.
```